matrix_io/vision/v1/vision.proto

```diff
@@ -17,6 +17,8 @@ syntax = "proto3";
 
 package matrix_io.vision.v1;
 
+import "google/protobuf/timestamp.proto";
```

Comment:
Perhaps add a comment here saying that this is installed with protoc. In case somebody has an issue getting this to work.